docs/docs/unreachable.md

```diff
@@ -0,0 +1,143 @@
+
+---
+title: Unreachable Strategy
+---
+
+# Unreachable Strategy
```

Comment:
I'd love to see a preamble that briefly explains what we understand by _unreachable task_. Something along the lines of

> *What is an unreachable task?*
>Sometimes tasks become unreachable from Marathon's perspective. The reason might be that the agent running the task gets disconnected from the Mesos master; due to a network partition, or because it is rebooted. Marathon might not even notice this, if [...]. If Marathon encounters a task being unreachable, it will [...]

Etc. if something like this is mentioned anywhere, probably moving it to the beginning would be helpful.